script_runner/frontend/src/ScriptResult.tsx

```diff
@@ -98,11 +98,23 @@ function getGridData(mergedData: MergedRowData[] | null) {
   return null;
 }
 
+// Helper to prevent XSS if displaying raw remote details/snippets
+function escapeHtml(unsafe: string | undefined | null): string {
```

Comment:
i don't think you need to escapeHtml here, since we render the regions, and error in the page as regular text

even if the api returned a html string we'd just dump the whole string `<div>whatever</div>` in the page not as html